prow/github/client.go

```diff
@@ -475,11 +538,13 @@ func (c *Client) GetCombinedStatus(org, repo, ref string) (*CombinedStatus, erro
 
 // getLabels is a helper function that retrieves a paginated list of labels from a github URI path.
 func (c *Client) getLabels(path string) ([]Label, error) {
+	var labels []Label
 	if c.fake {
-		return nil, nil
+		labels = append(labels, Label{Name: "lgtm", Color: "green"})
```

Comment:
@spxtr having test code inside prod is very very ugly. Why don't we follow the way fake clients are built in k8s, that is, by using a client interface?

Reply:
LOL I thought that I've removed this....

Reply:
I've tested locally with those test data removed:
```
iMac-Adm:label_sync lukasz$ git diff
diff --git a/prow/github/client.go b/prow/github/client.go
index 31de8e9a..690bf254 100644
--- a/prow/github/client.go
+++ b/prow/github/client.go
@@ -540,8 +540,6 @@ func (c *Client) GetCombinedStatus(org, repo, ref string) (*CombinedStatus, erro
 func (c *Client) getLabels(path string) ([]Label, error) {
        var labels []Label
        if c.fake {
-               labels = append(labels, Label{Name: "lgtm", Color: "green"})
-               labels = append(labels, Label{Name: "priority/P0", Color: "red"})
                return labels, nil
        }
        nextURL := strings.Join([]string{c.base, path}, "")
iMac-Adm:label_sync lukasz$ 
```

This is not needed for tests and can be removed any time:
```
iMac-Adm:label_sync lukasz$ make test
go test -race -cover $(go list ./... | grep -v "\/vendor\/")
ok  	k8s.io/test-infra/label_sync	1.074s	coverage: 32.7% of statements
iMac-Adm:label_sync lukasz$ 
```

Reply:
Mind opening a follow-up pr to remove them?

Reply:
I need to finish for today.
I'll open tomorrow unless anybody else does it until then.
It is just a super trivial PR to open 2 lines of code in a single file.

Reply:
OK I'm on it now